README.md

```diff
@@ -1,2 +1,56 @@
-# aurora
-Next generation tools
+# Aurora project #
+
+The main purpose of this subsystem is to unify one-liners approaches based on Ansible best practices. This one-liner is able to install Docker, download the specified image and create a new container for you. It will also create a desktop icon to start the container and launch the hand.
+
+## Before running the docker-deploy playbook ##
+
+Before setting up the docker container, ethercatinterface parameter for the hand needs to be discovered. In order to do so, after plugging the hand’s ethernet cable into your machine and powering it up, please run
+```shell
+sudo dmesg
+```
+command in the console. At the bottom, there will be information similar to the one below:
+```shell
+[490.757853] IPv6: ADDRCONF(NETDEV_CHANGE): enp0s25: link becomes ready
+```
+In the above example, ‘enp0s25’ is the ethercatinterface that is needed. 
+
+## How to run docker-deploy playbook ##
+
+Open a terminal with Ctrl+Alt+T and run:
+
+```bash
+bash <(curl -Ls bit.ly/run-ansible-sh) docker-deploy option1=value1 option2=value2 option3=value3
+```
+
+Possible options for the docker-deploy are:
+<pre>
+* product                   Name of the product (hand_e or hand_h)
+* image                     Name of the Docker hub image to pull
+* reinstall                 Flag to know if the docker container should be fully reinstalled (default: false)
+* name                      Name of the docker container
+* ethercatinterface         Ethercat interface of the hand
```

Comment:
please use words separated by underscore sign for variables names

Reply:
fixed in the last commit